### DISCUSSION TITLE
Getting permission from the user when exiting the Ballistica workspace

### DISCUSSION THREAD

**Discussion by HeyErfan** [efroemling/ballistica#680] [Ideas]
When editing a code in Ballistica's workspace, we prompt the user for reconfirmation to ensure that their exit from the workspace is intentional. This way, we ensure that any edited codes are saved and not lost in case of accidental exit.

**Comment by MatiasMGz** [comment]
Please add this. It has happened to me that i lost it accidentally! :(
